test/e2e/demos_test.go

```diff
@@ -39,6 +39,15 @@ var _ = Describe("Introduction", func() {
 		kubectlAndCheck("wait", "nncp", "vlan100", "--for", "condition=Available", "--timeout", "2m")
 	}
 
+	// Policies are not deleted as a part of the tutorial, so we need additional function here
+	cleanupConfiguration := func() {
+		deletePolicy("eth1")
+		deletePolicy("eth2")
```

Comment:
This are delete by `"delete", "nncp", "bond0-eth1-eth2"` although the deletePolicy failing if the policy is not there feels REALLY wrong.

Reply:
You are right